test/flink_test.go

```diff
@@ -65,6 +65,8 @@ func (s *CLITestSuite) TestFlinkStatement() {
 		{args: "flink statement delete my-statement --force --cloud aws --region eu-west-1", fixture: "flink/statement/delete.golden"},
 		{args: "flink statement list --cloud aws --region eu-west-1", fixture: "flink/statement/list.golden"},
 		{args: "flink statement list --cloud aws --region eu-west-1 -o yaml", fixture: "flink/statement/list-yaml.golden"},
+		{args: "flink statement list --cloud aws --region eu-west-1 --status COMPLETED", fixture: "flink/statement/list-completed.golden"},
+		{args: "flink statement list --cloud aws --region eu-west-1 --status PENDING", fixture: "flink/statement/list-pending.golden"},
```

Comment:
Let's test that the uppercase function works. This is also how we should recommend users to enter this in our examples.
```suggestion
		{args: "flink statement list --cloud aws --region eu-west-1 --status completed", fixture: "flink/statement/list-completed.golden"},
		{args: "flink statement list --cloud aws --region eu-west-1 --status pending", fixture: "flink/statement/list-pending.golden"},
```